images/win/scripts/Tests/Vsix.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "VsixTests" {
+    $toolset = Get-ToolsetContent
+    $requiredVsixs = $toolset.visualStudio.vsix
+
+    $allPackages = (Get-VSSetupInstance | Select-VsSetupInstance -Product *).Packages
+    $testCases = $requiredVsixs | ForEach-Object { @{ VsixId = $_.Id;  AllPackages = $allPackages }}
+
+    It "<VsixId>" -TestCases $testCases {

```

Comment:
let's update test name to "Extension <VsixId>" to make them more clear

Reply:
Name has been changed to "VSIX Extension <VsixId>"